### DISCUSSION TITLE
Swissup Ajaxpro Quickshop issues

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#61] [3. Integrations]
Hi There,

I installed and configured the Extension in my site. Also the quickshop popup showing on PLP. 

But the gallery thumbnails are not showing on popup in the media gallery.

Also in console I am getting below error : 
![image](https://user-images.githubusercontent.com/9527439/217244429-64cc5649-6984-4828-a597-c28070d02a7c.png)
![image](https://user-images.githubusercontent.com/9527439/217244589-de040ca7-9191-407b-9725-dbb64a7bee10.png)

Can you please help me on this. 

Thanks

**Comment by vovayatsyuk** [comment]
~~What is quickshop popup?~~ If you are using our ajaxpro module, please ensure that all modules are up to date.

**Comment by rutulritz** [comment]
We are using below versions : 

"swissup/ajaxpro": "^1.6",
"swissup/module-easytabs": "^1.11",
"swissup/module-marketplace": "^1.9",
"swissup/theme-frontend-breeze-evolution": "^1.2",

**Reply by vovayatsyuk** [reply]
You can check if you have any updates available using `composer outdated "swissup/*"` command

**Comment by rutulritz** [comment]
I upgraded all the swissup modules to latest version. Still thumbnails not showing and giving same error of 'PANZOOM' as I given screenshots in previous comment

**Reply by vovayatsyuk** [reply]
`"swissup/theme-frontend-breeze-evolution": "^1.2"` - this is the old version. Replace it will `^2.0` and run `composer update --W` then.

**Reply by vovayatsyuk** [reply]
Here is an upgrade guide: https://breezefront.com/upgrade

**Comment by rutulritz** [comment]
I upgraded modules and now below is the latest verisons: 

"swissup/ajaxpro": "^1.6",
"swissup/module-easytabs": "^1.11",
"swissup/module-marketplace": "^1.9",
"swissup/theme-frontend-breeze-evolution": "^2.1",

Still getting same errors as below:
![Screenshot from 2023-02-09 17-08-45](https://user-images.githubusercontent.com/9527439/217803915-7c1b94c5-433d-487e-ba40-feaf81267e05.png)

**Reply by vovayatsyuk** [reply]
Please share the link to your site so I could check it.

**Reply by rutulritz** [reply]
https://katespade.codilar.in/handbags/category/view-all.html

**Reply by vovayatsyuk** [reply]
One of your images in the json data doesn't have the required information to render the template. Do you have any idea why?

https://user-images.githubusercontent.com/306080/218026389-ca57821c-3328-4e23-a0ef-a2540e3ed947.mp4

**Reply by rutulritz** [reply]
where can I find this code ?

**Reply by rutulritz** [reply]
@vovayatsyuk I checked by printing json data in JS file and the thumbs are exists for the product on PLP 

![image](https://user-images.githubusercontent.com/9527439/218028606-b33aacf1-a1ee-4e9c-ac01-45a0ed2a7993.png)

**Reply by vovayatsyuk** [reply]
1. I just opened a product page and saw the error`Uncaught ReferenceError: full is not defined`. Here is a link https://katespade.codilar.in/market-spade-flower-jacquard-stripe-medium-tote-0k5000000-pxrub271.html

2. Regarding your error with panzoom, you need to install this module https://github.com/breezefront/module-breeze-integrations. It adds compatibility with swissup and few more modules.
